account_asset_management_xls/tests/test_asset_management_xls.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2009-2018 Noviat.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import odoo.tests.common as common
+from odoo import tools
+from odoo.modules.module import get_resource_path
+
+
+class TestAssetManagementXls(common.TransactionCase):
+
+    def _load(self, module, *args):
+        tools.convert_file(self.cr, module,
+                           get_resource_path(module, *args),
+                           {}, 'init', False, 'test',
+                           self.registry._assertion_report)
+
+    def setUp(self):
+        super(TestAssetManagementXls, self).setUp()
+
+        self._load('account', 'test', 'account_minimal_test.xml')
```

Comment:
do we really need to load all this stuff for each test?
If this is a base class it means that for every test running we reload the whole thing again and again if we have more than one test.

SavepointCase + setUpClass to the rescue :)

Reply:
@simahawk 
Can you make a PR that optimises this ? 
I'll review and test